connaisseur/admission_review.py

```diff
@@ -60,6 +62,9 @@ def get_admission_review(
             status["message"] = msg
         review["response"]["status"] = status
 
+    if warnings:
```

Comment:
`if warnings is not None:` (if an empty list is passed explicitly, we should honor it). Alternatively, the default should be the empty list (and then the `if` condition is fine).